task/common/machine/storage.go

```diff
@@ -19,6 +21,8 @@ import (
 	"github.com/rclone/rclone/fs/operations"
 	"github.com/rclone/rclone/fs/sync"
 
+	"github.com/sirupsen/logrus"
```

Comment:
Use the logger instead. its not a big thing but we already have a formatter

Reply:
forget what I said. Logrus is overriden by us 🤦

Reply:
It already uses our formatter as a side efect of `init()` here: 🤢 
https://github.com/iterative/terraform-provider-iterative/blob/64c828042d75a6edb3f8571aa73c48886189c9ae/iterative/utils/logger.go#L34

Reply:
Yes thats right